Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Oceananigans"
 uuid = "9e8cae18-63c1-5223-a75c-80ca9d6e9a09"
 authors = ["Climate Modeling Alliance and contributors"]
-version = "0.74.0"
+version = "0.74.1"
```

Comment:
I'm not 100% clear on the versioning we do. Since this adds a feature should it be a minor version bump? Or are those only for breaking changes?

Reply:
We've been bumping the minor version for API changes while bumping just the patch release for new features or bug fixes.